src/pages/GetStarted/__tests__/updateDevfileMetadata.spec.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2018-2021 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { updateDevfileMetadata } from '../updateDevfileMetadata';
+import { safeDump } from 'js-yaml';
+
+describe('Update devfile metadata', () => {
+
+  it('should return a devfile V1 as is', () => {
+    const devfile = {
+      apiVersion: '1.0.0',
+      metadata: {
+        generateName: 'wksp-',
+      },
+    };
+
+    const targetDevfile = updateDevfileMetadata(devfile);
+
+    expect(targetDevfile).toEqual(devfile);
+  });
+
+  it('should return a devfile V2 with a custom metadata', () => {
+    const attributes = {
+      'dw.metadata.annotations': {
+        'che.eclipse.org/devfile-source': safeDump({
+          custom: {},
+        }),
+      },
+    };
+
+    let devfile = getV2Devfile();
+
+    expect(devfile.metadata.attributes).toEqual(undefined);
+
+    devfile = updateDevfileMetadata(devfile);
+
+    expect(devfile.metadata.attributes).toEqual(attributes);
+  });
+
+  it('should return a devfile V2 with a sample metadata', () => {
+    const metadata = getMetadata();
+    const attributes = {
+      'dw.metadata.annotations': {
+        'che.eclipse.org/devfile-source': safeDump({
+          sample: {
+            registry: metadata.registry,
+            name: metadata.displayName,
```

Comment:
well, PR description has name (`name: 'apache-camel-springboot'` which is not display name), but here in tests you use display name as name, it's confusing

Reply:
I have fixed it:
```name: metadata.displayName``` -> ```displayName: metadata.displayName```